pydrive2/test/README.rst

```diff
@@ -40,15 +41,30 @@ Run tests locally
       and click **Create** to proceed with a default client name. At **Credentials**
       screen find a list of your **OAuth 2.0 Client IDs**, click download icon in
       front of your OAuth client id created previously. You should be prompted to
-      download **client_secret_xxx_.json** file.
-    - **cd** to PyDrive project root
-      directory to create **configs** folder and copy downloaded **.json** file into
-      it. Rename the copied file to **client_secrets.json**.
+      download `client_secret_xxx_.json` file.
```

Comment:
looks you need to apply :code: before inline code block in RST - https://stackoverflow.com/questions/10870719/inline-code-highlighting-in-restructuredtext

Reply:
thanks, it works!

Reply:
👍